tests/Integration/IntegrationTest.php

```diff
@@ -28,6 +28,12 @@ public static function dataIntegrationTests(): iterable
         }
 
         yield [__DIR__ . '/data/eloquent-builder-l10.php'];
+
+        if (! version_compare(LARAVEL_VERSION, '11.0.0', '>=')) {
+            return;
+        }
+
+        yield [__DIR__ . '/data/model-casts-l11.php'];
```

Comment:
For Laravel 12

```suggestion
        if (version_compare(LARAVEL_VERSION, '11.0.0', '>=')) {
            yield [__DIR__ . '/data/model-casts-l11.php'];
        }
```

Reply:
I could do that, but this was the existing pattern there which is what I followed:

https://github.com/larastan/larastan/blob/2ac60334eb7e5d693df9a9150bd6798be680f99c/tests/Integration/IntegrationTest.php#L26-L30

Reply:
All right.